AustrianAddress/AustrianAddress.xml

```diff
@@ -23,60 +23,359 @@
   <type value="Address" />
   <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Address" />
   <derivation value="constraint" />
-  <differential>
+  <snapshot>

```

Comment:
Same issue as in #26, the whole snapshot representation for the AustrianAddress was added in this pull request.
Makes the code review more difficult than it otherwise would have been.
Not an error per se, just something I would add in a separate commit independently of these pull requests.

Reply:
Same as in #3 - This is because of the tools we are currently using - both Forge and the IG publisher need the snapshot to properly compute the effective profile structure.
Nevertheless we created a separate issue for this, see #30.